test_ui.py

```diff
@@ -281,6 +287,20 @@ def __init__(self):
         self.current_jpeg = 0
 
         self.id = self.device.getDeviceInfo().getMxId()
+        self.bootloader_version = "library"
+        try:
+            self.bootloader_version = self.device.getBootloaderVersion().toStringSemver()
+        except: pass
```

Comment:
Lets change the semantics a bit - let `bootloader_version` denote what BL version was flashed to the device (it might be that upon a next reboot, device wouldn't boot BL, this might be the case for PoE devices where we want to perform everything/most over USB for Test stage)

Also, in some cases we flash boot header - that would preferably be denoted as well (we can do just flashRead of 512B and store as is)

So:
 - which BL version was flashed (if it was flashed)
 - which type of BL (there are two, USB and NETWORK)
 - And first 512B of flash as is (raw)